tests/integration/golang/helpers/client.go

```diff
@@ -148,3 +148,39 @@ func (c HttpClient) doRequest(httpMethod string, uri string, response interface{
 
 	return nil
 }
+
+// DoGetRequestNoUnmarshalling executes GET request to the uri, returning
+// responses body or error
+func (c HttpClient) DoGetRequestNoUnmarshalling(uri string) ([]byte, error) {
```

Comment:
we have `DoStreamRequest` won't it work?

Reply:
I'll look

Reply:
good call, will switch to that